shell/platform/darwin/ios/framework/Source/FlutterDartVMServicePublisher.h

```diff
@@ -14,7 +14,7 @@
 - (instancetype)init NS_UNAVAILABLE;
 + (instancetype)new NS_UNAVAILABLE;
 
-@property(nonatomic, retain, readonly) NSURL* url;
+@property(nonatomic, strong, readonly) NSURL* url;
```

Comment:
Nit: I thought from the last discussion that we were moving toward not putting `strong`?

Reply:
You're right, I just saw a stray retain and didn't think about it.  Removing.